app/views/errors/internal_server_error.html.erb

```diff
@@ -0,0 +1,17 @@
+<div class="jumbotron">
+  <h1>Internal Server Error</h1>
+  <p>Please try again in a few moments. 
+  If you continue to see this error, please
+  <a href="<%= support_url %>">contact support</a>.
+  </p>
+
+
+</div>

```

Comment:
Fix the formatting of the html (maybe just remove extra newlines between `</p>` and `</div>`